core/src/main/java/com/hcl/ecomm/core/listeners/SimpleResourceListener.java

```diff
@@ -16,8 +16,8 @@
 package com.hcl.ecomm.core.listeners;
 
 import org.apache.sling.api.SlingConstants;
-import org.osgi.framework.Constants;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
```

Comment:
@DeepaliAgrwl Why do we have this file ?

Reply:
removed the file